packages/js/src/components/SEMrushRelatedKeyphrasesModalContent.js

```diff
@@ -117,14 +118,14 @@ export default function RelatedKeyphraseModalContent( props ) {
 			) }
 
 			{ getUserMessage( props ) }
-
-			<SEMrushKeyphrasesTable
-				keyphrase={ keyphrase }
-				relatedKeyphrases={ relatedKeyphrases }
-				countryCode={ countryCode }
-				renderAction={ renderAction }
-				data={ response }
-			/>
+			<Root>
```

Comment:
You should pass the wanted context. I.e. RTL, or you run the risk of this not being compatible

Reply:
I should do it in premium too

Reply:
What do you mean? You already render inside the root there.